<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeckleQGISDialogBase</class>
 <widget class="QDockWidget" name="SpeckleQGISDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>651</height>
   </rect>
  </property>

  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>

        
        <item row="0" column="1">
          <layout class="QHBoxLayout" name="streamListButtons">
          </layout>
        </item>

        <item row="1" column="0">
         <widget class="QLabel" name="streamListLabel">
          <property name="text">
           <string>Stream</string>
          </property>
         </widget>
        </item>

        <item row="1" column="1">
          <layout class="QHBoxLayout" name="streamListButtons" stretch="20,1">
            <item>
              <widget class="QComboBox" name="streamList"/>
            </item>
            <item>
            <widget class="QPushButton" name="streams_add_button">
              <property name="text">
              <string> </string>
              </property>
            </widget>
            </item>
            <item>
              <widget class="QPushButton" name="streams_remove_button">
                <property name="text">
                <string> </string>
                </property>
                  <property name="geometry">
                    <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>10</width>
                      <height>10</height>
                    </rect>
                  </property>
              </widget>
            </item>
          </layout>
        </item>

        <item row="2" column="0">
         <widget class="QLabel" name="streamBranchLabel">
          <property name="text">
           <string>Branch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="streamBranchDropdown"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="commitLabel">
          <property name="text">
           <string>Commit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="commitDropdown"/>
        </item>
        
        <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
         </layout>
        </item>

        
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="sendModeButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="receiveModeButton">
            <property name="text">
             <string>Receive</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>

        <item row="6" column="1">
         <widget class="QComboBox" name="layerSendModeDropdown"/>
        </item>



        <item row="7" column="1">
         <widget class="QListWidget" name="layersWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::Fixed</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>


        <item row="8" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout"  stretch="1,1">
            <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
              <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
              </property>
            </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="saveLayerSelection">
                <property name="enabled">
                  <bool>true</bool>
                </property>
                <property name="text">
                  <string>Save current layer selection</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>

        <item row="9" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout"  stretch="1,1">
            <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
              <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
              </property>
            </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="setMapping">
                <property name="enabled">
                  <bool>true</bool>
                </property>
                <property name="text">
                  <string>Apply transformations on Send</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>


        <item row="10" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout"  stretch="1,1">
            <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
              <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
              </property>
            </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="crsSettings">
                <property name="enabled">
                  <bool>true</bool>
                </property>
                <property name="text">
                  <string>Set project center on Send/Receive</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>

        <item row="11" column="0">
         <widget class="QLabel" name="messageLabel">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="messageInput">
          <property name="placeholderText">
           <string>Sent XXX objects from QGIS</string>
          </property>
         </widget>
        </item>

        

        <item row="12" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
          <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
          </spacer>
          </item>

          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string> SEND</string>
            </property>
           </widget>
          </item>

          <item>
          <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
          </spacer>
          </item>

         </layout>
        </item>

        
        <item row="13" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
            
            <item>
              <widget class="QPushButton" name="reloadButton">
                <property name="enabled">
                <bool>true</bool>
                </property>
                <property name="text">
                <string>Refresh</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="closeButton">
                <property name="enabled">
                  <bool>true</bool>
                </property>
                <property name="text">
                  <string>Close</string>
                </property>
              </widget>
            </item>

          </layout>
        </item>

       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
